ui/pages/Project/reducers.js

```diff
@@ -167,6 +167,8 @@ export const updateIndividuals = values => (dispatch, getState) => {
     action = `save_individuals_table/${values.uploadedFileId}`
   } else if (values.delete) {
     action = 'delete_individuals'
+  } else if (values.workspaceName) {
```

Comment:
Can you make `uploadedFileId`, `delete`, and `workspaceName` constants and use them wherever they're being defined originally? I know that `workspaceName` comes from a field on the uploaded Gregor metadata form but I haven't figured out yet where `uploadedFileId` comes from.

Reply:
I think fully refactoring a helper function used in several different place plus updating a whole lot of forms unrelated to this change to use those constants is outside the scope of this PR,  but I am happy to add comments to make it clearer

Reply:
I agree that's out of scope, comments about where those fields come from would help too